.github/ISSUE_TEMPLATE/issue.yaml

```
@@ -31,8 +33,41 @@ body:
       description: Contact and name info for key parties for the project.
       placeholder: |
         E.g.:
-        Casey Greene, PI, casey@greenelab.com, @caseygreene on slack
-        John Doe, project lead, john.doe@fake.com, @johndoe on slack
+        Casey Greene, PI, casey@greenelab.com, @caseygreene on Slack
+        John Doe, project lead, john.doe@fake.com, @johndoe on Slack
+    validations:
+      required: true
+
+  - type: dropdown
+    id: category
+    attributes:
+      label: Type of support
+      description: Which of these best describe the type of support you need?
+      options:
+        - Short/medium-term project (ongoing)
+        - Debugging/troubleshooting (one-off)
+        - Consulting/education (one-off)
+        - Other
+      multiple: true
+      default: 0
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Describe the specific support you need from SET. E.g.:
```

I worry that the reader might think they need to answer all the questions on this list, despite it saying "E.g.". I get that we might not get all the information we need if we soften the language, but IMHO I'd rather people submit "incomplete" requests than be put off making a request all.

Totally willing to change this, but here's a first shot:

```suggestion
        Describe the support you need from SET. Feel free to put what you need in your own words, but here are some common questions:
```